tests/src/Unit/OgAccessHookTest.php

```diff
@@ -22,25 +21,38 @@ public function setUp() {
   }
 
   /**
+   * Tests that an entity which is not a group or group content is ignored.
+   *
    * @dataProvider operationProvider
    */
   public function testNotContentEntity($operation) {
     $entity = $this->prophesize(EntityInterface::class);
     $access = og_entity_access($entity->reveal(), $operation, $this->user->reveal());
+
+    // An entity which is not a group or group content should always return
+    // neutral, since we have no opinion over it.
     $this->assertTrue($access->isNeutral());
   }
 
   /**
+   * Tests that an administrator with 'administer group' permission has access.
+   *
    * @dataProvider operationProvider
    */
   public function testGetEntityGroups($operation) {
     $this->user->hasPermission(OgAccess::ADMINISTER_GROUP_PERMISSION)->willReturn(TRUE);
-    $user_entity_access = og_entity_access($this->entity->reveal(), $operation, $this->user->reveal());
+    $user_entity_access = og_entity_access($this->groupContentEntity->reveal(), $operation, $this->user->reveal());
+
+    // @todo This is strange, 'view' is not part of the operations supplied by
+    //   ::operationProvider(). And why would a group administrator be allowed
+    //   access to all operations, except 'view'? Shouldn't this also return
+    //   'allowed'?
```

Comment:
this is related to the incorrect crud operation To permission name bug/ feature we have.

The gist of things is - OG's hook_entity_access() cares only about non-view operations. If it's a `view` we are neutral about it. The reason is that for example node access should be determined by the node grants, and not on the fly.

So this

```php
$user_entity_access = og_entity_access($this->entity->reveal(), $operation, $this->user->reveal());
```

should probably change to something like
```
$permissions_name = OgPermissionsHandler::getPermissionFromEntityOperation($entity, 'view');
$user_entity_access = og_entity_access($this->entity->reveal(), $permissions_name, $this->user->reveal());
```

It can probably be a follow up, as it's out of the scope of this PR. So maybe open an issue instead of the `@todo`?